kuma/wiki/content.py

```diff
@@ -681,7 +681,11 @@ def __iter__(self):
                 # treat it as a manual override by the author and make
                 # that value be the ID.
                 if (None, 'name') in attrs:
-                    attrs[(None, u'id')] = attrs[(None, 'name')]
+                    # Sanitize the "name" attribute with self.slugify to
+                    # prevent the injection of spaces (which are illegal
+                    # for the "id" attribute) or any of the non-URL-safe
+                    # characters listed above.
+                    attrs[(None, u'id')] = self.slugify(attrs[(None, 'name')])
```

Comment:
If we always dynamically get the value of `name` at runtime, this should be safe.